api/bazel/repository_locations.bzl

```diff
@@ -88,4 +88,15 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-08-17",
         use_category = ["api"],
     ),
+    com_github_apache_skywalking_data_collect_protocol = dict(
+        project_name = "SkyWalking API",
+        project_desc = "SkyWalking's language independent model and gRPC API Definitions",
+        project_url = "https://github.com/apache/skywalking-data-collect-protocol",
+        version = "8.1.0",
```

Comment:
8.2.0 is out at https://github.com/apache/skywalking-data-collect-protocol/releases/tag/v8.2.0. Do you want to bump here or do as a follow up?

Reply:
8.1.0 and 8.2.0 are the same for the tracing format of the language agent. I think it is fine. 
SkyWalking wouldn't break the tracing format for a long time to keep all language agents compatible and LTS.

Reply:
If this is an envoy project requirement, @wbpcode the 8.1.0->8.2.0 should not break anything you wrote :)